contracts/LpSugar.vy

```diff
@@ -353,6 +357,17 @@ def _byData(_data: address[3], _account: address) -> Lp:
   token0: IERC20 = IERC20(pool.token0())
   token1: IERC20 = IERC20(pool.token1())
   gauge_alive: bool = self.voter.isAlive(gauge.address)
+  acc_balance: uint256 = pool.balanceOf(_account)
```

Comment:
I'm a bit concerned of the gas usage now that we're introducing these extra calls, should we skip fully the balance and claimable calculations if the `_account` is `0x0...00` ?